app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt

```diff
@@ -173,20 +175,10 @@ class ImageViewerActivity : AppCompatActivity() {
   }
 
   private fun flickGestureListener(): FlickGestureListener {
-    val contentHeightProvider = object : ContentSizeProvider {
-      override fun heightForDismissAnimation(): Int {
-        return imageView.zoomedImageHeight.toInt()
-      }
-
-      // A positive height value is important so that the user
-      // can dismiss even while the progress indicator is visible.
-      override fun heightForCalculatingDismissThreshold(): Int {
-        return when {
-          imageView.drawable == null -> resources.getDimensionPixelSize(
-              R.dimen.media_image_height_when_empty)
-          else -> imageView.visibleZoomedImageHeight.toInt()
-        }
-      }
+    val contentSizeProvider = ContentSizeProvider2 {
+      // A non-zero height is important so that the user can dismiss even
+      // the image is unavailable and the progress indicator is visible.
+      maxOf(dip(240), imageView.zoomedImageHeight.toInt())
```

Comment:
On a second thought, `imageView.zoomedImageHeight() ?: dip(240)` might be easier to understand. Can change if you want.

Reply:
Does zoomedImageHeight() return a nullable?

Reply:
No, not yet

Reply:
Not yet? o_O

Reply:
Heh I meant it could be changed to do that

Reply:
Is that something we want? I'm mostly confused on why the comment is here if it's not something currently happening

Reply:
Na, let's ignore this comment. :)